### ISSUE TITLE
Wont work?

### ISSUE THREAD

**Issue by eyemiru** [nating/whatsapp-stats#1]
Tried it today, wont to anything.

**Comment by nating** [comment]
Hi @eyemiru, thanks for checking the repo out. Could you give me any more information about what isn't working? At the moment, only chats exported from iOS are working because chats from different operating systems are exported in a different format and need to be parsed in different ways.

**Comment by eyemiru** [comment]
Hi @nating, yes it was from Android. Can you give me an example how to parse them or what need to be done?

**Comment by nating** [comment]
Of course. Let me get back to you later today, as I am working at the moment. 👌👍

**Comment by eyemiru** [comment]
Thank you :)
